plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestDynamicTable.java

```diff
@@ -448,4 +448,27 @@ public void testQuotesInColumnName()
         assertEquals(extractPql(dynamicTable, TupleDomain.all()), expectedPql);
         assertEquals(dynamicTable.getTableName(), tableName);
     }
+
+    @Test
+    public void testQueryOptions()
+    {
+        String tableName = realtimeOnlyTable.getTableName();
+        String tableNameWithSuffix = tableName + REALTIME_SUFFIX;
+        String query = """
+                SET skipUpsert='true';
+                SET useMultistageEngine='true';
+                SELECT FlightNum
+                FROM %s
+                LIMIT 50;
+                """.formatted(tableNameWithSuffix);
+        DynamicTable dynamicTable = buildFromPql(pinotMetadata, new SchemaTableName("default", query), mockClusterInfoFetcher, TESTING_TYPE_CONVERTER);
+        String expectedPql = """
+                SET skipUpsert = 'true';
+                SET useMultistageEngine = 'true';
```

Comment:
Does Pinot accept sting literals for numerics options? e.g. `numReplicaGroupsToQuery`

Can we add an integration test in BasePinotConnectorSmokeTest?

Reply:
added test for query option.